clouddriver-titus/src/test/groovy/com/netflix/spinnaker/clouddriver/titus/TitusExceptionSpec.groovy

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.clouddriver.titus
+
+import spock.lang.Specification
+
+class TitusExceptionSpec extends Specification {
+
+  def "should set retryable to false on invalid arguments"() {
```

Comment:
Maybe add a small test here of a message that does not start with `INVALID_ARGUMENT`.